AsyncDisplayKit/ASDisplayNode+Deprecated.h

```diff
@@ -15,6 +15,14 @@
 @interface ASDisplayNode (Deprecated)
 
 /**
+ * @abstract The name of this node, which will be displayed in `description`. The default value is nil.
+ * 
+ * @deprecated Deprecated in version 2.0: Use .debugName instead. This value will display in 
+ * results of the -asciiArtString method (@see ASLayoutElementAsciiArtProtocol).
+ */
+@property (nullable, nonatomic, copy) NSString *name;
```

Comment:
does this need to have an NS_DEPRECATED or something?  There might be an ASDK version of that annotation too.